src/widget/wtracktableview.h

```diff
@@ -74,7 +75,13 @@ class WTrackTableView : public WLibraryTableView {
     void slotReloadCoverArt();
 
   private:
-    void sendToAutoDJ(bool bTop);
+    enum AutoDJSendLoc {
+        SEND_TOP,
+        SEND_BOTTOM,
+        SEND_REPLACE,
+    };
```

Comment:
Prefer to use a strongly typed C++11 scoped enumeration:
enum class AutoDJSendLoc {
TOP,
BOTTOM,
REPLACE
};

-> AutoDJSendLoc::TOP, ...

http://en.cppreference.com/w/cpp/language/enum